Engine/DataFeeds/CollectionSubscriptionDataSourceReader.cs

```diff
@@ -79,7 +80,8 @@ public IEnumerable<BaseData> Read(SubscriptionDataSource source)
                 {
                     default:
                     case SubscriptionTransportMedium.Rest:
-                        reader = new RestSubscriptionStreamReader(source.Source, source.Headers);
+                        var isliveMode = source.IsLiveMode != null && source.IsLiveMode.Value;
```

Comment:
For example, this class already has a `_isLiveMode` class member variable we can use here.